README.md

```diff
@@ -9,8 +9,14 @@ Sorry if the name causes any confusion but this gem doesn't currently output Sas
     % gem install less2sass
 
 ### Usage
+#### Single file conversion
+
     % less2sass inputfile.less [options]
 
+#### Multiple file conversion
+
+    % less2sass *.scss
```

Comment:
Please change to `% less2sass *.less [options]`. Also, add the following example `% less2sass inputfile1.less inputfile2.less [options]`